src/scss/styles.scss

```diff
@@ -123,6 +123,18 @@ body {
     &.layout_frontend {
         background-color: #ecf0f5;
     }
+
+    &.full-width {
```

Comment:
Just add `:not(&.layout_frontend)` or something to that affect to exclude the frontend layout.